docs/pages/distribution/publishing-websites.mdx

```diff
@@ -309,33 +310,25 @@ In case you want to change the header for hosting add the following config for `
 
 <Step label="1">
 
-Start by initializing a new git repository in your project. If this is already done, skip this step.
+Start by initializing a new git repository in your project and configuring it to push to a GitHub repository. If you are already syncing your changes with a GitHub repository, skip this step.
 
-If not then you'll want to run the following command in your project's root directory:
+Create a repository on the GitHub website. Then, run the following commands in your project's root directory:
 
-<Terminal cmd={['$ git init']} />
+<Terminal cmd={['$ git init', '$ git remote add origin https://github.com/username/expo-gh-pages.git' ]} />
 
-</Step>
-
-<Step label="2">
-
-Add the GitHub repository as a `remote` in your local git repository.
-
-<Terminal cmd={['$ git remote add origin https://github.com/username/expo-gh-pages.git']} />
-
-Running the above command configures git know which repository you want to push your source code to.
+The above commands initialize a new git repository and configure git to know which repository on GitHub to push your source code to.
 
 </Step>
 
-<Step label="3">
+<Step label="2">
 
 Install the `gh-pages` package as a `dev-dependency` in your project:
 
 <Terminal cmd={['$ yarn add -D gh-pages']} />
 
 </Step>
 
-<Step label="4">
+<Step label="3">
 
 Configure your project to be deployed to a subdomain with the **baseUrl** feature in the **app.json** Set its value to the string `/{repo-name}`.
```

Comment:
```suggestion
To deploy the project, configure it to a subdomain with the [`baseUrl`](/versions/latest/config/app/#baseurl) property in [app config](/workflow/configuration/). Set its value to the string `/repo-name`.
```

(Unrelated to this PR) This sentence has a typo. `baseUrl` is a property not a a feature in app config. Also, let's drop using `{}` as it indicates the user also needs to them in the configuration.